_includes/default-tutorial.html

```diff
@@ -0,0 +1,476 @@
+
+      <li class="annotation" id="a18">The embedding layer is specified as an AllenNLP <code>TextFieldEmbedder</code> which represents a general way of turning tokens into tensors. (Here we know that we want to represent each unique word with a learned tensor, but using the general class allows us to easily experiment with different types of embeddings.)</li>
+
+      <li class="annotation" id="a19">Similarly, the encoder is specified as a general <code>Seq2SeqEncoder</code> even though we know we want to use an LSTM. Again, this makes it easy to experiment with other sequence encoders.</li>
+
+      <li class="annotation" id="a20">Every AllenNLP model also expects a <code>Vocabulary</code>, which contains the namespaced mappings of tokens to indices and labels to indices.</li>
+
+      <li class="annotation" id="a21">Notice that we have to pass the vocab to the base class constructor.</li>
+
+      <li class="annotation" id="a22">The feed forward layer is not passed in as a parameter, but is constructed by us. Notice that it looks at the encoder to find the correct input dimension and looks at the vocabulary (and, in particular, at the label -> index mapping) to find the correct output dimension.</li>
+
+      <li class="annotation" id="a23">The last thing to notice is that we also instantiate a <code>CategoricalAccuracy</code> metric, which we'll use to track accuracy during each training and validation epoch. Because of the dependency injection, that's all we have to do to construct the model.</li>
+
+      <li class="annotation" id="a24">Next we need to implement <code>forward</code>, which is where the actual computation happens. Each Instance in your dataset will get (batched with other instances and) fed into forward. The forward method expects tensors as input, and it expects their names to be the names of the fields in your Instance. In this case we have a sentence field and (possibly) a labels field, so we'll construct our <code>forward</code> accordingly:</li>
+
+      <li class="annotation" id="a25">AllenNLP is designed to operate on batched inputs, but different input sequences have different lengths. Behind the scenes AllenNLP is padding the shorter inputs so that the batch has uniform shape, which means our computations need to use a mask to exclude the padding. Here we just use the utility function <code>get_text_field_mask</code>, which returns a tensor of 0s and 1s corresponding to the padded and unpadded locations.</li>
+
+      <li class="annotation" id="a26">We start by passing the <code>sentence</code> tensor (each sentence a sequence of token ids) to the <code>word_embeddings</code> module, which converts each sentence into a sequence of embedded tensors.</li>
```

Comment:
This was the annotation that had an extra open `<code>` tag instead of a closing `</code>` tag. I fixed it here, but FYI.